### PR TITLE
DRILL-6286: Fixed incorrect reference to shutdown in drillbit.log

### PR DESCRIPTION
**Pull request by dvjyothsna**
@vrozov , @ilooner Please review

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java [hunk 1]
```diff
@@ -86,6 +86,9 @@
   private final StatusThread statusThread;
   private final Lock isEmptyLock = new ReentrantLock();
   private final Condition isEmptyCondition = isEmptyLock.newCondition();
+  private boolean isShutdownTriggered = false;
```
Comment (vrozov) [inline exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java]: Is this boolean necessary? Can you delay getting `isEmptyCondition` till shutdown is requested and use it in place of `isShutdownTriggered`?

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java [hunk 2]
```diff
@@ -212,19 +218,29 @@ private boolean areQueriesAndFragmentsEmpty() {
     return queries.isEmpty() && runningFragments.isEmpty();
   }
 
+  /**
+   * Check if there any new queries or fragments that are added after the shutdown is triggered
+   */
+  private boolean areNewQueriesOrFragmentsAdded() {
+    return runningFragments.size() > numOfRunningFragments || queries.size() > numOfRunningQueries;
```
Comment (vrozov) [inline exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java]: This condition is not reliable. What if some fragments exited and some were added? The total number may still be less than the number of fragments when the shutdown was requested.

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java [hunk 3]
```diff
@@ -212,19 +218,29 @@ private boolean areQueriesAndFragmentsEmpty() {
...
   /**
    * A thread calling the {@link #waitToExit(boolean)} method is notified when a foreman is retired.
    */
   private void indicateIfSafeToExit() {
     isEmptyLock.lock();
     try {
-      logger.info("Waiting for "+ queries.size() +" queries to complete before shutting down");
-      logger.info("Waiting for "+ runningFragments.size() +" running fragments to complete before shutting down");
+      if (isShutdownTriggered) {
+        logger.info("Waiting for "+ queries.size() +" queries to complete before shutting down");
```
Comment (vrozov) [inline exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java]: Use slf4j smart logging.

**Comment by dvjyothsna** [comment]
@vrozov please review

**Comment by vrozov** [comment]
+1

**Pull request closed without merging**